amqp/src/main/scala/akka/stream/alpakka/amqp/javadsl/AmqpSink.scala

```diff
@@ -17,6 +17,12 @@ object AmqpSink {
     Sink.fromGraph(new AmqpSinkStage(settings))
 
   /**
+    * Java API: Creates an [[AmqpSink]] that accepts [[OutgoingMessage]] elements.
+    */
+  def createReplyTo(settings: AmqpSinkSettings): akka.stream.javadsl.Sink[OutgoingMessage, NotUsed] =
```

Comment:
Duplicate the docs from `AmqpReplyToStage` here to make it clear how it differs from `create`.

Also, maybe we should fail the "regular" sink if `replyTo` is set on an `OutgoingMessage`, so that if someone mixes them up it will at least fail as fast as possible? (Ofc optimal would be to have two different types so doing it wrong wouldn't even compile, but to do that we'd have to break the existing method and I don't think we should do that)